server/models/services/UserService.js

```diff
@@ -480,6 +516,22 @@ const removeUserRole = async (userId, roleName, { transaction }) => {
   });
 };
 
+/**
+ * @param {number} userId - id of the User that the vendor will be added to
+ * @param {number} vendorId - id of the Vendor that the User will be associated with
+ * @param {object} options
+ * @param {*} options.transaction - Sequelize transaction
+ * @returns {Promise<*>}
+ */
+const addUserVendor = async (userId, vendorId, { transaction }) => {
```

Comment:
Yeah I am not in love with the name but wasn't sure where best to move the term from "vendor" to "company" since a "vendor" attached to a person is their "company".